Story/Episode 1/un_010540.csv

```diff
@@ -1,5 +1,5 @@
-name01#0,"""？？？"""
+name01#0,"""???"""
```

Comment:
Do not change the ???, the strings from the English patch will take care of them